src/Database/Dialect/PostgresDialectTrait.php

```diff
@@ -129,6 +129,24 @@ protected function _transformFunctionExpression(FunctionExpression $expression)
             case 'NOW':
                 $expression->name('LOCALTIMESTAMP')->add([' 0 ' => 'literal']);
                 break;
+            case 'DATE_ADD':
+                $expression
+                    ->name('')
+                    ->type(' + INTERVAL')
+                    ->iterateParts(function ($p, $key) {
+                        if ($key === 1) {
+                            $p = sprintf("'%s'", $p);
```

Comment:
is it not possible to pass that as a bound parameter? I fear this could become a sql injection vector